src/Bijectors.jl

```diff
@@ -160,6 +208,35 @@ end
 _logabsdetjac_dist(d::LKJCholesky, x::Cholesky) = logabsdetjac(bijector(d), x)
 _logabsdetjac_dist(d::LKJCholesky, x::AbstractVector) = logabsdetjac.((bijector(d),), x)
 
+"""
+    logpdf_with_trans(d::Distribution, x, transform::Bool)
+
+If `transform` is `false`, `logpdf_with_trans` calculates the log probability
+density function (logpdf) of distribution `d` at `x`.
+
+If `transform` is `true`, the logpdf is calculated after transforming `x` using
+the constrained-to-unconstrained bijector for distribution `d`. Specifically,
```

Comment:
I find the first sentence a bit difficult to understand correctly. One could be lead to think of something like `logpdf(d, link(d, x))`. Or if one doesn't think of that, then one might wonder why it makes a difference whether you transform or not. Is there some alternative phrasing for it? Maybe something about not accounting for the Jacobian term?

Reply:
I had an original draft that wrote 'transforming both `x` and `d` using the constrained-to-unconstrained bijector', then I took it out because I wasn't sure if 'transforming a distribution' was clear. Would that be better?

It is true that `logpdf_with_trans(d, x, true) = logpdf(transform(d), link(d, x))`.

Reply:
I think that's much clearer. Would you also be up for adding a sentence saying this is essentially just ignoring the Jacobian term? I think for anyone with background in this sort of stuff that could be illuminating.

Reply:
`transform=true` does use the Jacobian, whereas `transform=false` doesn't (because it doesn't change variables):

```julia
julia> using Bijectors

julia> logpdf_with_trans(LogNormal(), ℯ, true)
-1.4189385332046727

julia> logpdf_with_trans(LogNormal(), ℯ, false) - logabsdetjac(bijector(LogNormal()), ℯ)
-1.4189385332046727
```

I can definitely add a note to this effect, though!

Reply:
Should be better now :)